src/pages/teamMembers.js

```diff
@@ -0,0 +1,245 @@
+import React, { Fragment } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import GridList from '@material-ui/core/GridList';
+import GridListTile from '@material-ui/core/GridListTile';
+import GridListTileBar from '@material-ui/core/GridListTileBar';
+import Modal from '@material-ui/core/Modal';
+import Backdrop from '@material-ui/core/Backdrop';
+import Zoom from '@material-ui/core/Zoom';
+
```

Comment:
We can import all of the components in this fashion : { GridList, Container, Modal....} from '@material-ui/core'

Reply:
Okay